src/index.js

```diff
@@ -115,14 +126,10 @@ class Anvil {
       },
       { dataType: DATA_TYPE_JSON },
     )
-    const {
-      data: {
-        generateEtchSignURL,
-      },
-    } = data
+
     return {
       statusCode,
-      url: generateEtchSignURL,
+      url: data?.data?.generateEtchSignURL,
       errors,
     }
```

Comment:
Snuck in a tiny reformatting change here so that a potential error wouldn't be thrown. Doesn't affect rest of code. Hope that's fine with you @newhouse! 🙂

Reply:
All good, thx

Reply:
Look at you with your optional chaining!!!

Reply:
![yo dawg i heard you like data](https://user-images.githubusercontent.com/69169/93624963-073bed80-f996-11ea-905e-8bd0db718d4a.png)